__tests__/healthcareProfessional.test.ts

```diff
@@ -81,6 +83,494 @@ describe('createHealthcareProfessional', () => {
+        
+        expect(createdProfessional).toBeFalsy()
+        expect(createProfessionalResult.body?.errors).toBeDefined()
+        expect(createProfessionalResult.body?.errors[0].extensions.errors[0]).toBeDefined()
+        expect(createProfessionalResult.body?.errors[0].extensions.errors.length).toBeGreaterThanOrEqual(3)
+
+        const errorNameEn = createProfessionalResult.body?.errors[0].extensions.errors[0]
+        const errorNameJa = createProfessionalResult.body?.errors[0].extensions.errors[1]
+        const errorAbbreviation = createProfessionalResult.body?.errors[0].extensions.errors[2]
+
+        expect(errorNameEn.field).toBe('degrees[0].nameEn')
+        expect(errorNameEn.errorCode).toBe(ErrorCode.CONTAINS_INVALID_CHARACTER)
+        expect(errorNameEn.httpStatus).toBe(400)
+
+        expect(errorNameJa.field).toBe('degrees[0].nameJa')
+        expect(errorNameJa.errorCode).toBe(ErrorCode.CONTAINS_INVALID_CHARACTER)
+        expect(errorNameJa.httpStatus).toBe(400)
+
+        expect(errorAbbreviation.field).toBe('degrees[0].abbreviation')
+        expect(errorAbbreviation.errorCode).toBe(ErrorCode.CONTAINS_INVALID_CHARACTER)
+        expect(errorAbbreviation.httpStatus).toBe(400)
+    })
+
+    test('failing: throws an error if degree is empty', async () => {
+        const createFailingHealthcareProfessional = {
+            query: createHealthcareProfessionalMutation,
+            variables: {
+                input: {
+                    facilityIds: sharedFacilityIds,
+                    names: [testData.generateLocalizedNameInput()],
+                    degrees: [],
+                    specialties: [testData.generateSpecialty()],
+                    spokenLanguages: testData.generateSpokenLanguages(),
+                    acceptedInsurance: [testData.generateAcceptedInsurance()]
+                }
+            }
+        } as gqlMutation<CreateHealthcareProfessionalInput>
+
+        const createProfessionalResult = await request(gqlApiUrl).post('/').send(createFailingHealthcareProfessional)
+        const createdProfessional
+                     = createProfessionalResult.body.data.createHealthcareProfessional as HealthcareProfessional
+        
+        expect(createdProfessional).toBeFalsy()
+        expect(createProfessionalResult.body?.errors).toBeDefined()
+        expect(createProfessionalResult.body?.errors[0].extensions.errors[0]).toBeDefined()
+        expect(createProfessionalResult.body?.errors[0].extensions.errors.length).toBeGreaterThanOrEqual(1)
+
+        const error = createProfessionalResult.body?.errors[0].extensions.errors[0]
+
+        expect(error.field).toBe('degrees')
+        expect(error.errorCode).toBe(ErrorCode.REQUIRED)
+        expect(error.httpStatus).toBe(400)
+    })
+})
+
+describe('deleteHealthcareProfessional', () => {
+    test('deletes a new healthcare professional', async () => {
```

Comment:
I had a merge conflict in this file, I hope I merged correct? This part is what is on main now

Reply:
looks ok!